gtsfm/utils/metrics.py

```diff
@@ -100,13 +98,14 @@ def compute_rotation_angle_metrics(wRi_list: List[Optional[Rot3]], gt_wRi_list:
     """
     errors = []
     for (wRi, gt_wRi) in zip(wRi_list, gt_wRi_list):
-        errors.append(comp_utils.compute_relative_rotation_angle(wRi, gt_wRi))
-    return compute_errors_statistics(errors)
+        if wRi is not None and gt_wRi is not None:
+            errors.append(comp_utils.compute_relative_rotation_angle(wRi, gt_wRi))
+    return GtsfmMetric("rotation_averaging_angle_deg", np.array(errors))
```

Comment:
do we need the `np.array(errors)`? isn't `List[float]` an acceptable datatype as input?

Reply:
bump

Reply:
marking as resolved, thanks